src/slides/slides.css

```diff
@@ -555,6 +546,13 @@ body:lang(en) {
   color: unset;
 }
 
+.reveal .slides section section .version {
+  color: var(--grey);
+  right: 50px;
+  font-size: 13px;
+  visibility: visible;
```

Comment:
Is this really useful? I think `visible` is the default.

Reply:
It's not useful per se, but it will help me hide the version info on non-title slides. 😉